creativecloud/blocks/sidenav/sidenav.js

```diff
@@ -0,0 +1,38 @@
+import { createTag, loadScript } from '../../scripts/miloUtils.js';
+
+await import('../../deps/sidenav.js');
+loadScript('/creativecloud/deps/merch-spectrum.min.js');
```

Comment:
`merch-spectrum.min.js` should be imported before `sidenav.js` to be on the safe side.
also why not`import` but `loadScript`?

Reply:
isn't there a risk that it's imported twice? if not why using loadScript at all in milo?

Reply:
no imports are always done once, it is safer ;)

Reply:
import loads ecma script modules,  whereas loadScript can load arbitrary javascript.